src/tools/pmreorder/pmreorder.py

```diff
@@ -9,6 +9,9 @@
 import markerparser
 import sys
 import reorderengines
+import signal
+
+signal.signal(signal.SIGINT, lambda signum, frame: exit(signum))
```

Comment:
is it safe to replace SIGINT handler like this?

Can you add a comment? It's really non-obvious to me as to what precisely this does. Normally OS-level kill doesn't propagate to children. You'd have to use process group. I guess it's a python-ism.